examples/cogvideo/train_cogvideox_lora.py

```diff
@@ -0,0 +1,1594 @@
+# Copyright 2024 The HuggingFace Team.
```

Comment:
@a-r-r-o-w since this is going to be a canonical script, I think we should have a test similar to https://github.com/huggingface/diffusers/blob/main/examples/dreambooth/test_dreambooth_flux.py